### PR TITLE
Additional changes for GeoFire 2.0.0

### PR DESCRIPTION
**Pull request by mcdonamp**
Podspec changes for the version bump
Removed `setup.sh` and replaced with Podfile
README updates

### CODE REVIEW

**Comment by TimKreger-IA** [comment]
The readme also needs updating to reflect the changes. Would you like me to do that?

**Comment by mcdonamp** [comment]
@TimKreger-IA I believe I updated it here, but I'd appreciate the second set of eyes to make sure I didn't miss anything.

**Comment by TimKreger-IA** [comment]
@mcdonamp  Sorry I swear I looked at the changes 3 times and didn't see the README update!!

**Comment by mcdonamp** [comment]
It's all good, I was mostly just confused if I hadn't uploaded the changes correctly.

**Comment by matijse** [comment]
@mcdonamp Regarding the readme, in the Quickstart, maybe it is better to not create the reference to the root, but to a child path? Since you wouldn't want to store the geofire data in the root...

There is already a note explaining that this is possible, but it is easily overlooked.

File: GeoFire.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -24,14 +24,18 @@
...
 		38FC6CB71CFBF907002AB722 /* GFQueryTest.m in Sources */ = {isa = PBXBuildFile; fileRef = FB8C884B196D93E10061AC0F /* GFQueryTest.m */; };
+		D871159D1D093CFF00D74E95 /* FirebaseAnalytics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D87115971D093CFF00D74E95 /* FirebaseAnalytics.framework */; };
+		D871159E1D093CFF00D74E95 /* FirebaseDatabase.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D87115981D093CFF00D74E95 /* FirebaseDatabase.framework */; };
+		D871159F1D093CFF00D74E95 /* FirebaseInstanceID.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D87115991D093CFF00D74E95 /* FirebaseInstanceID.framework */; };
+		D87115A01D093CFF00D74E95 /* GoogleInterchangeUtilities.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D871159A1D093CFF00D74E95 /* GoogleInterchangeUtilities.framework */; };
+		D87115A11D093CFF00D74E95 /* GoogleSymbolUtilities.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D871159B1D093CFF00D74E95 /* GoogleSymbolUtilities.framework */; };
+		D87115A21D093CFF00D74E95 /* GoogleUtilities.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = D871159C1D093CFF00D74E95 /* GoogleUtilities.framework */; };
```
Comment (jdimond) [inline GeoFire.xcodeproj/project.pbxproj]: Are these added by the CocoaPods? If so, can we remove them again?

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ At its heart, GeoFire simply stores locations with string keys. Its main
 benefit however, is the possibility of querying keys within a given geographic
 area - all in realtime.
 
-GeoFire uses the [Firebase](https://www.firebase.com/?utm_source=geofire-objc) database for
+GeoFire uses the [Firebase](https://www.firebase.google.com/?utm_source=geofire-objc) database for
```
Comment (jdimond) [inline README.md]: This URL is wrong (no www)

**Comment by psopher** [comment]
Thanks again for doing all this!

For what it's worth, it would be very helpful for at least me if you didn't wait for the java version before deploying the cocoapod for this one.

**Comment by jdimond** [comment]
LGTM. Thanks for putting this together.

**Pull request merged**